ymcatwincities/openy
# [OYS-26] - Implement prototype of directories for media

Make sure these boxes are checked before asking for review of your pull request - thank you!

If there is a new feature or this is a bug fix - use 8.x-2.x branch. We'll tag for release if the bug is critical asap or tag for release next bug fix release until critical issue arrived.

## Steps for review

- Create or edit News Post.
- [ ] Ensure image field makes possible to open an image library to upload a new file or select from already existing ones.
- Go to Content > Media.
- [ ] Ensure an image library is opened, and gives possibility to filter medias by directories.
